care/facility/api/viewsets/facility.py

```diff
@@ -32,22 +31,33 @@ class FacilityFilter(filters.FilterSet):
 
 class FacilityQSPermissions(DRYPermissionFiltersBase):
     def filter_queryset(self, request, queryset, view):
+        search_text = request.query_params.get("search_text")
```

Comment:
This was changed to use the Django ORM, can you merge the latest master, and raise again.